### ISSUE TITLE
RuntimeError: shape '[0, 8, 4096, 320]' is invalid for input of size 2621440

### ISSUE THREAD

**Issue by zakinp** [ShivamShrirao/diffusers#98]
a problem occur when i use the up-to-date "train_dreambooth.py" to finetune the model.

**Comment by zakinp** [comment]
firstly, i convert a .ckpt model to diffusers. then i use the script train the converted model and this problem occur

**Comment by damienrj** [comment]
I am also getting this issue, but am letting the code download the model.

**Comment by zakinp** [comment]
> 

yeah, it seems like a bug. This error also occur when i convert the waifu-diffusion 1.3 model in huggingface to train through dreambooth.

**Comment by zakinp** [comment]
> I am also getting this issue, but am letting the code download the model.

yeah, it seems like a bug. This error also occur when i convert the waifu-diffusion 1.3 model in huggingface to train through dreambooth.

**Comment by jaworek** [comment]
I'm getting similar error. I updated `main` branch to latest commit and also applied patch from #102. In both cases I still get the same error. I'm trying to run it on MacBook with M1 Pro and 32GB of RAM.

<img width="652" alt="image" src="https://user-images.githubusercontent.com/22678765/198987255-d361d72c-2681-456a-a7c6-b907b3452ca6.png">
